packages/python-packages/api-stub-generator/apistub/nodes/_docstring_parser.py

```diff
@@ -1,27 +1,22 @@
-from typing import OrderedDict
+from collections import OrderedDict
```

Comment:
Can we isolate just this change into separate PR to resolve breakage in repo? Currently this issue is holding us back from deploying any changes in APIView since parser and APIView are deployed as single entity.